app/src/main/java/io/neurolab/main/NFBServer.java

```diff
@@ -26,11 +32,30 @@
     private LinkedList<Task> tasks;
     private long lastTimestamp;
 
+    public Config getConfig() {
+        return config;
+    }
```

Comment:
@jddeep Everything looks good. Just rearrange the code. Getters and setters should be below.